Create postgresql-expiring-passwd.md (#94)
percona-platform/portal-doc: docs/advisors/checks/postgresql-expiring-passwd.md

# PostgreSQL roles with expiring passwords

## Description

> This will check roles that have expiration dates on passwords. This is helpful in preventing applications accounts from failing due to expired passwords thus preventing interruptions to services offered.

Reviewer: ```suggestion
This advisor checks roles that have expiration dates on passwords. This is helpful in preventing applications accounts from failing due to expired passwords thus preventing interruptions to services offered.
```

> This will check roles that have expiration dates on passwords. This is helpful in preventing applications accounts from failing due to expired passwords thus preventing interruptions to services offered.

Reviewer 2: ```suggestion
This check verifies roles that have expiration dates on passwords. This ensures that application accounts do not fail due to expired passwords and prevents possible interruptions to the services offered.
```

> If a password will expire in 10 days or less, an Error flag is raised. If the password will expire after 10 days, a Warning flag is raised. 

Reviewer: ```suggestion
If a password expires in 10 days or less,  an Error flag is raised. If the password expires in more than 10 days, a Warning flag is raised. 
```

## Resolution

> It is advisable to extend the password expiration for the role in question if policy allows for it.

Reviewer: ```suggestion
Consider extending the password expiration for the role in question if your policy allows for it.
```

> You can extend a password expiration with the following syntax:

Reviewer 2: ```suggestion
Use the following syntax to  extend password expiration:
```
